data/teams_fixture.csv

```diff
@@ -1,6 +1,33 @@
 team_id,franchiseId,teamName,abbreviation,Stadium,link

```

Comment:
Will approve merge after we figure out whether to keep the fixture files or not tomorrow.